doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -37,10 +37,10 @@ following Features tabs defined here.
 | Conformance                            | X             | Kubernetes conformance test |
 | None                                   | X             | Kubernetes mainstream features |
 | Feature:ComprehensiveNamespaceDraining | X             | Namespaces should always delete fast (ALL of 100 namespaces in 150 seconds) |
-| Feature:CrossNamespacePodAffinity      |               | Should verify ResourceQuota with cross namespace pod affinity scope using scope-selectors |
-| Feature:PodPriority                    | X             | Verify ResourceQuota's priority class scope against a pod with different priority class |
+| Feature:[CrossNamespacePodAffinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#namespace-selector)|               | Should verify ResourceQuota with cross namespace pod affinity scope using scope-selectors |
+| Feature:[PodPriority](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)| X             | Verify ResourceQuota's priority class scope against a pod with different priority class |
 | Feature:ScopeSelectors                 | X             | Verify ResourceQuota with terminating scopes through scope selectors |
```

Comment:
@collivier @CsatariGergely Is "https://kubernetes.io/docs/concepts/policy/resource-quotas/#quota-scopes" a suitable link as this is about "terminating" scopes?

Reply:
In my reading this is not about scope selectors.